manual/cli.rst

```diff
@@ -1,4 +1,100 @@
+| -v         | --verbose            | Enable verbose messages                                                                                                       |
++------------+----------------------+-------------------------------------------------------------------------------------------------------------------------------+
+|            | --version            | Display version information                                                                                                   |
++------------+----------------------+-------------------------------------------------------------------------------------------------------------------------------+
+| --bw       | --black-and-white    | Draw maps in black and white                                                                                                  |
++------------+----------------------+-------------------------------------------------------------------------------------------------------------------------------+
+
+
+Options valid only for generate
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+| Short     | Long                       | Description                                                                                          |
++===========+============================+======================================================================================================+
+| -r FILE   | --rivers=FILE              | produce a map of rivers, after the option it expects the name of the file where to generate the map  |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+| --gs      | --grayscale-heightmap      | produce a grayscale heightmap                                                                        |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+|           | --ocean_level=N            |  elevation cut off for sea level [default = 1.0]                                                     |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+|           | --temps #/#/#/#/#/#        |  Provide alternate ranges for temperatures. [default = .126/.235/.406/.561/.634/.876]                |
++-----------+----------------------------+------------------------------------------------------------------------------------------------------+
+|           | --humidity #/#/#/#/#/#/#   | Provide alternate ranges for humidities. [default = .059/.222/.493/.764/.927/.986/.998]              |
```

Comment:
Since this is the manual, these two could use some rudimentary explanation (must be in [0, 1]; names of the biomes in order of the given numbers and maybe even an explanation of the numbers themselves).
As is, this isn't comprehensible to me, and I should already know what these mean.